core/src/main/java/com/sequenceiq/cloudbreak/converter/spi/InstanceMetaDataToCloudInstanceConverter.java

```diff
@@ -61,9 +66,21 @@ public List<CloudInstance> convert(List<InstanceGroupDto> instanceGroups, String
     public List<CloudInstance> convert(Collection<InstanceMetadataView> instanceMetadataViews, StackView stack) {
         List<CloudInstance> cloudInstances = new ArrayList<>();
         DetailedEnvironmentResponse environment = environmentClientService.getByCrnAsInternal(stack.getEnvironmentCrn());
+        logConvertDetails(environment.getCloudPlatform(), String.format("Converting to cloudInstances based on input param instanceDataViews: %s",
+                Joiner.on(",").join(instanceMetadataViews.stream()
+                                .map(imd -> String.format("[name: %s, instance group name: %s]", imd.getInstanceName(), imd.getInstanceGroupName()))
+                                .collect(Collectors.toSet()))));
```

Comment:
tip: you could  use `Collectors#joining`

Reply:
I would not change this for now, I'd remove this logging preferably after RCA is done

Reply:
https://jira.cloudera.com/browse/CB-18710